c2corg_ui/templates/image/index.html

```diff
@@ -0,0 +1,40 @@
+<%inherit file="../base.html"/>
+<%namespace file="../helpers/common.html" import="show_title"/>
+<%namespace file="../helpers/list.html" import="add_doctype_selector"/>
+
+<%block name="pagetitle"><title ng-bind="mainCtrl.page_title('Images')">${show_title('Images')}</title></%block>
+
+<%block name="moduleConstantsValues">
+  module.value('mapFeatureCollection', null);
+  module.value('advancedSearchFilters', {
+    'date': {'type': 'date'},
+    'act': {'type': 'list'},
+    'oalt': {'type': 'range'}
+  });
```

Comment:
I will have to index these fields in ElasticSearch. Which fields to you need? `activities, categories, elevation, date_time`? Maybe also `image_type`?